### ISSUE TITLE
Final Fantasy XIV - Launcher doesn't start - "Cannot complete version check"

### ISSUE THREAD

**Issue by yaomtc** [sonic2kk/steamtinkerlaunch#761]
## System Information
- SteamTinkerLaunch version: 12.0
- Distribution: Arch Linux
- Installation Method: Package manager / AUR

## Issue Description
When attempting to start the game with steamtinkerlaunch, I get an error: "Cannot complete version check". I've tried it with a new prefix and that doesn't help.

## Logs

[steamtinkerlaunch.log](https://github.com/sonic2kk/steamtinkerlaunch/files/10965066/steamtinkerlaunch.log)

**Comment by sonic2kk** [comment]
Hmm, this is strange, I can't reproduce the problem. I'm also running Arch Linux.

I haven't played FFXIV in a long time but I re-installed it to test. I was able to open the launcher and attempt to log in using STL v12.0 and master. I had wondered if maybe the issue would be related to STL v12 not using the Steam Linux Runtime, which is an issue fixed in master, but both v12.0 worked and so did latest master, with SLR enabled and disabled.

I was able to open the launcher and log in, after which the launcher told me I did not have an active subscription so I couldn't advance any further. I could not reproduce this issue under normal circumstances, with and without SteamTinkerLaunch, so I'm kind of stumped here.

<hr>

I was only able to create a similar error to the one you described, and only under what I would consider expected circumstances: by turning off WiFi while the game was opening, so it could contact the Steam keyserver and launch the game, but _disconnected_ while the launcher dialog was up saying it was checking the launcher version. I was able to reliably show the error using this method with and without SteamTinkerLaunch. The only way I would reproduce the issue with STL was to enable the "Block Internet" option in the Game Menu.

In other words, I was only able to re-create this issue whenever there was a network issue, and not when using STL alone.

![image](https://user-images.githubusercontent.com/7917345/224996111-8e5f7ad1-dfd7-4e29-b716-9b91b9ad8b56.png)

This is as far as I can get to test, and this appears with and without STL likely because I don't have an FFXIV subscription, so I would also say this is expected. I also am assuming that the error you're seeing is expected to appear when first opened, in other words I should have seen it before I got to this point anyway.

![image](https://user-images.githubusercontent.com/7917345/225097902-371919e1-2836-4d93-b6a6-10ca696d5c82.png)

<hr>

I can't get further than logging in so unless the error appears after logging in and so on, it seems I can't re-create the issue. As you specifically mention in the title that the launcher doesn't start I would assume this error happens whenever the white dialog for the launcher opens stating that it's checking the version.

The log you provided doesn't seem to be for a game launch, there is no logging about the game exe launch, but if the launcher is opening I would guess STL is running the Proton command properly.

I don't see how SteamTinkerLaunch could be causing an issue here to begin with, as all it does is export some environment variables when asked to and so on, and then launch the game with the same command the Proton script would (or a best-guess at a native execution, when it comes to native Linux games). So I don't think this is a SteamTinkerLaunch issue.
